mmv1/templates/terraform/examples/dns_managed_zone_quickstart.tf.erb

```diff
@@ -0,0 +1,50 @@
+#[START dns_domain_tutorial]
+# to setup a web-server
+resource "google_compute_instance" "default" {
+  name         = "<%= ctx[:vars]['dns_compute_instance'] %>"
+  machine_type = "g1-small"
+  zone         = "us-central1-b"
+
+  boot_disk {
+    initialize_params {
+      image = "debian-cloud/debian-9"
+    }
+  }
+
+  network_interface {
+    network = "default"
+       access_config {
+           // Ephemeral public IP
+   }
+  }
+  metadata_startup_script = "sudo apt-get update && sudo apt-get install apache2 -y && echo \'<!doctype html><html><body><h1>Hello World!</h1></body></html>\' > /var/www/html/index.html"
```

Comment:
Hmm now we are getting the error `The symbol "'" is not a valid escape sequence selector.`
What if we made the startup script a variable and just passed it from the `terraform.yaml`?

Reply:
Let me try to replace them with `\"`. I am hoping this should do.

Reply:
I have referred other erb samples and used EOF to update the code snippet. Thank you @c2thorn